setup.py

```diff
@@ -16,34 +17,12 @@ def run_tests(self):
         sys.path.insert(0, 'lib')
         pytest.main(self.test_args)
 
+os.environ['SKIP_GENERATE_AUTHORS'] = '1'
```

Comment:
why do you skip it ?
The AUTHORS file is not managed in this repo, so you can let pbr to include it ...